11x-new-kafka-roller.md

```diff
@@ -0,0 +1,263 @@
+# Kafka Roller 2.0
+
+## Current situation
+
+The Kafka Roller is an internal Cluster Operator component that's responsible for coordinating the rolling restart or reconfiguration of Kafka pods when:
+- non-dynamic reconfigurations needs to be applied
+- update in Kafka CR is detected
+- a TLS certificate is renewed
+- pods have been [manually annotated](https://strimzi.io/docs/operators/latest/full/deploying#rolling_pods_manually_alternative_to_drain_cleaner) by the user for controlled restarts
+- pod is stuck and has a pending update (e.g. not running with the desired version or configuration)
+- Kafka broker is unresponsive to Kafka Admin connections
+
+A pod is considered stuck if it is in one of following states:
+- `CrashLoopBackOff`
+- `ImagePullBackOff`
+- `ContainerCreating`
+- `Pending` and `Unschedulable`
+
+
+### Known bugs
+
+- KafkaRoller takes a long time to reconcile combined nodes if they are all in Pending stateMore information can be found [here](https://github.com/strimzi/strimzi-kafka-operator/issues/9426).
```

Comment:
You can still keep separate lines within a bullet point, I think this formatting should work. Also added some extra context.
```suggestion
- KafkaRoller takes a long time to reconcile combined nodes if they are all in Pending state.
  A fix for this is not possible without making non-trivial changes to the KafkaRoller.
  More information can be found [here](https://github.com/strimzi/strimzi-kafka-operator/issues/9426).
```